docker-compose.yaml

```diff
@@ -171,11 +171,11 @@ services:
       KEYCLOAK_CLIENTS_0_PERMISSIONS_0_RESOURCE_SERVER_ID: operate-api
       KEYCLOAK_CLIENTS_0_PERMISSIONS_0_DEFINITION: read:*
     healthcheck:
-      test: [ "CMD", "curl", "-f", "http://localhost:8084/actuator/health" ]
+      test: [ "CMD", "curl", "-f", "http://localhost:8082/actuator/health" ]
```

Comment:
Not sure why `8084` was used before 🤷🏻. We also use `8082` here: https://github.com/camunda/web-modeler/blob/16a41d3c94fecb20d8665b59bd16dc9414ebfb0b/docker-compose.sm.yml#L100

Reply:
Doesn't make a difference on my machine. Container still starts as unhealthy

Reply:
Huh, that's weird! Can you check the output of this command?
```sh
docker exec -it identity curl -f http://localhost:8082/actuator/health
```

Reply:
Is reported as `{"status":"UP"}

Reply:
Nevermind, I messed something up when checking out the branch